File: tests/e2e/fixtures/unsigned_integers/mysql/kotlin/src/main/kotlin/com/example/db/com/example/db/models/UnsignedValues.kt
package com.example.db.models

data class UnsignedValues(
    val id: java.math.BigInteger,
    val u8Val: Short,
    val u16Val: Int,
    val u24Val: Long,
    val u32Val: Long,
    val u64Val: java.math.BigInteger
)

File: tests/e2e/fixtures/unsigned_integers/mysql/rust/src/db/models/unsigned_values.rs
#[derive(Debug, sqlx::FromRow)]
pub struct UnsignedValues {
    pub id: u64,
    pub u8_val: u8,
    pub u16_val: u16,
    pub u24_val: u32,
    pub u32_val: u32,
    pub u64_val: u64,
}
